app/(afterLogin)/(timesheetInformation)/timesheet/settings/approvals/workFlow/approvalSetting/page.tsx

```
@@ -48,17 +48,18 @@ const ApprovalSetting: React.FC<any> = () => {
         onSuccess: () => {
           NotificationMessage.success({
             message: 'Success',
-            description: 'Approver created successfully',
+            description: 'Approver created successfully'
+
```

ESLint

[error] Replace `⏎` with `,`

(prettier/prettier)